### PR TITLE
Template for quality indicators 

### PR DESCRIPTION
**Pull request by dgarijo**
Following the metadata schema I am working in https://github.com/EVERSE-ResearchSoftware/schemas

### CODE REVIEW

**Comment by dgarijo** [comment]
This template also adds one example based on the list in https://docs.google.com/spreadsheets/d/1lLLEsB7QZF32DeljFIDvCA70bvgG5QhsAZ9gZ6mCfLc/edit?usp=sharing

**Comment by dgarijo** [comment]
Please note: the w3id of the indicator schema is not yet operational

**Comment by dgarijo** [comment]
I ended adding a quality dimension template for referring to the different dimensions/characteristics we will be using in the RSQKit. I added one dimension taken from the reference model

**Review by anenadic — APPROVED** [review]
This all looks good to me - the only suggestion I have to is explictly say "software quality indicators" and "software quality indicator dimensions" just so it is clear what we are talking aboout here to anyone coming to RSQKit for the first time.

I have provided some suggestions for this @dgarijo - otherwise, happy merging as far as I am concerned 🎉. @shoaibsufi may want to look at this as well.

File: _data/quality_dimensions.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Conforms to: https://w3id.org/everse/rsqd/0.0.1
+# Summary of the metadata fields for dimensions:
```
Comment (anenadic) [inline _data/quality_dimensions.yaml:2]:
```suggestion
# Summary of the metadata fields for software quality indicator dimensions: 
```

**Comment by dgarijo** [comment]
Nice catch, thanks!

File: _data/quality_indicators.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# qualityDimension: Research Software quality dimension associated with the indicator (see https://w3id.org/everse/rsqi for a full list) 
+# version: If the indicator has a version (some ISO standards may evolve indicators), it should be indicated here
+# source: The standard or reference document where the indicator was first proposed
+# status: States whether the indicator is still used or not (deprecated, obsolete, active, etc.)
+# created: Date of creation of the indicator
+
+- identifier: Do1 # Note: everse will mint an URI for this indicator under: https://w3id.org/everse/indicators/{identifier}
+  description: >
+    Software includes clear contribution instructions, software purpose on project website, feedback channels, and version control practices.
+  contact: Daniel Garijo # see CONTRIBUTORS.yaml to declare contacts
```
Comment (anenadic) [inline _data/quality_indicators.yaml:22]:
```suggestion
  contact: Daniel Garijo # see CONTRIBUTORS.yml to declare contacts
```

File: _data/quality_indicators.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  name: Contribution Process and Project Purpose
+  keywords:
+    - contribution guidelines
+    - feedback channel
+    - software purpose
+  qualityDimension:
+    - documentation # This is an identifier (see dimensions.yaml)
```
Comment (anenadic) [inline _data/quality_indicators.yaml:29]:
```suggestion
    - documentation # This is an identifier (see quality_dimensions.yaml)
```

**Pull request merged**